src/lib/dialog/dialog.spec.ts

```diff
@@ -40,14 +41,15 @@ describe('MatDialog', () => {
   let dialog: MatDialog;
   let overlayContainer: OverlayContainer;
   let overlayContainerElement: HTMLElement;
+  let focusMonitor: FocusMonitor;
 
   let testViewContainerRef: ViewContainerRef;
   let viewContainerFixture: ComponentFixture<ComponentWithChildViewContainer>;
   let mockLocation: SpyLocation;
 
   beforeEach(fakeAsync(() => {
     TestBed.configureTestingModule({
-      imports: [MatDialogModule, DialogTestModule],
+      imports: [MatDialogModule, DialogTestModule, A11yModule],
```

Comment:
Importing the `A11yModule` shouldn't be necessary here, it's imported by the `MatDialogModule` already.

Reply:
That's true, but I thought since we monitor the button independently from the dialog in a test, I think being explicit is better.